integration/raft/config_test.go

```diff
@@ -1078,6 +1078,77 @@ var _ = Describe("EndToEnd reconfiguration and onboarding", func() {
 			Expect(resp.Status).To(Equal(common.Status_SUCCESS))
 		})
 
+		When("an evicted node is added back while it's offline", func() {
+			It("can start with correct new raft id", func() {
+				o1 := network.Orderer("orderer1")
+				o2 := network.Orderer("orderer2")
+				o3 := network.Orderer("orderer3")
+
+				By("Waiting for them to elect a leader")
+				findLeader(ordererRunners)
+
+				By("Creating a channel testchannel")
+				network.CreateChannel("testchannel", o1, peer)
+
+				assertBlockReception(map[string]int{
+					"testchannel":   0,
+					"systemchannel": 1,
+				}, []*nwo.Orderer{o1, o2, o3}, peer, network)
+
+				By("Killing the first orderer")
+				ordererProcesses[0].Signal(syscall.SIGTERM)
+				Eventually(ordererProcesses[0].Wait(), network.EventuallyTimeout).Should(Receive())
+
+				// We need to wait for stabilization, as we might have killed the leader OSN.
+				By("Waiting for the channel to stabilize after killing the orderer")
+				assertBlockReception(map[string]int{
+					"testchannel":   0,
+					"systemchannel": 1,
+				}, []*nwo.Orderer{o2, o3}, peer, network)
+
```

Comment:
could you please add one step here to assert/wait for ActiveNode metrics to be 2? this would allow us to deterministically reproduce the Node ID error, and assure that #1367 fixes it

Reply:
#1367 is still in open.
Consider the severity, do you think we can reduce to one-round remove-add in the test case, and then update after #1367 got merged?